controllers/common.go

```diff
@@ -720,10 +732,45 @@ func GetPodSecurityContext(namespace string, sourcePVCName string, c client.Clie
 func podHasPVCName(pod *corev1.Pod, pvcName string) *corev1.Pod {
 
 	for _, vol := range pod.Spec.Volumes {
-		if vol.PersistentVolumeClaim != nil && vol.PersistentVolumeClaim.ClaimName == pvcName {
+		if vol.PersistentVolumeClaim != nil &&
+			vol.PersistentVolumeClaim.ClaimName == pvcName {
 			return pod
 		}
 	}
+	return nil
+}
+
+func getContainerSecurityContext(pod *corev1.Pod) *corev1.SecurityContext {
 
+	containerSC := corev1.SecurityContext{}
+	for _, container := range pod.Spec.Containers {
+		if container.SecurityContext != nil {
+			containerSC = *container.SecurityContext
+			return &containerSC
+		}
+	}
 	return nil
 }
+
+func buildPodSecurityContext(sc *corev1.SecurityContext, pod *corev1.Pod) *corev1.PodSecurityContext {
```

Comment:
```suggestion
func buildMoverPodSecurityContext(sc corev1.SecurityContext, podSC corev1.PodSecurityContex) *corev1.PodSecurityContext {
```
Makes parameter not a pointer so parameter is a copy.
Pass in just security contexts as opposed to the entire pod definition